src/components/CharacterList/index.tsx

```diff
@@ -1,6 +1,43 @@
 import React from "react"
-import "./styles.scss"
 
-export default function CharacterList() {
-  return <div>CharacterList</div>
+interface ICharacter {
+  name: string
+  id: number
+  gender: string
+  spacies: string
+  origin: string
+  type: string
+  image: string
+  location: string
+  episodes: object[]
+}
+interface CharacterListProps {
```

Comment:
we're adding `I` to beginning of the interfaces. change it to `ICharacterListProps`